--- setup.py.old	2023-06-16 15:55:53.305829396 -0500
+++ setup.py	2023-06-16 15:56:11.316897488 -0500
@@ -5,6 +5,6 @@
 if __name__ == "__main__":
     setup(
         name="etelemetry",
-        version=versioneer.get_version(),
+        version="0.3.0",
         cmdclass=versioneer.get_cmdclass(),
     )
--- etelemetry/__init__.py.old	2023-06-16 16:17:29.146397433 -0500
+++ etelemetry/__init__.py	2023-06-16 16:17:48.555486467 -0500
@@ -1,4 +1,4 @@
 from .client import get_project, check_available_version, BadVersionError
 
 from . import _version
-__version__ = _version.get_versions()['version']
+__version__ = "0.3.0"
