--- scripts/print_qiime_config.py.org	2014-09-09 16:50:16.043866672 -0500
+++ scripts/print_qiime_config.py	2014-09-09 16:50:35.594825852 -0500
@@ -267,7 +267,7 @@
         proc = Popen(command,shell=True,universal_newlines=True, \
                          stdout=PIPE,stderr=STDOUT)
         stdout = proc.stdout.read()
-        version_string = stdout.strip().split('Python')[-1].strip()
+        version_string = stdout.strip().strip().split('Python')[-1].strip().split('::')[0].strip()
         try:
             if version_string[-1]=='+':
                 version_string = version_string[:-1]
