--- Makefile.Cuda.old	2020-12-16 20:11:16.423784842 -0600
+++ Makefile.Cuda	2020-12-16 20:13:05.602232096 -0600
@@ -15,7 +15,7 @@
 CPP = g++
 UNAME := $(shell uname)
 
-TARGETS = 52 60 61 70
+TARGETS = 35 37 60 70 72 75
 CUDA_TARGETS=$(foreach target,$(TARGETS),-gencode arch=compute_$(target),code=sm_$(target))
 
 $(shell ./link_cuda.sh)
@@ -100,16 +100,16 @@
 
 ifeq ($(CONFIG),FDEBUG)
 	OPT =-O0 -g3 -Wall -DDOCK_DEBUG
-    CUDA_FLAGS = -G -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11
+    CUDA_FLAGS = -G -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11 -ccbin $(CC)
 else ifeq ($(CONFIG),LDEBUG)
 	OPT =-O0 -g3 -Wall 
-	CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11
+	CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11 -ccbin $(CC)
 else ifeq ($(CONFIG),RELEASE)
 	OPT =-O3
-	CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11
+	CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11 -ccbin $(CC)
 else
 	OPT =
-    CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11
+    CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11 -ccbin $(CC)
 endif
 
 # ------------------------------------------------------
