--- configure.old	2024-06-20 23:58:14.536931503 -0500
+++ configure	2024-06-21 00:02:41.510768219 -0500
@@ -265,9 +265,8 @@
 cleanpath = [ entry for entry in oldpath if 'anaconda' not in entry.lower() ]
 if not oldpath == cleanpath:
   report ('WARNING: Anaconda removed from PATH to avoid conflicts\n\n')
-path = os.pathsep.join(cleanpath)
-
-os.environ['PATH'] = path
+#path = os.pathsep.join(cleanpath)
+#os.environ['PATH'] = path
 
 
 
