--- src/neper_t/net_ori/net_ori_uniform/net_ori_uniform4.cpp	2018-07-25 07:37:57.000000000 +0000
+++ src/neper_t/net_ori/net_ori_uniform/net_ori_uniform4.cpp.new	2018-10-12 21:13:21.785687300 +0000
@@ -21,7 +21,7 @@
       double Eij, *fij = ut_alloc_1d (3);
       net_ori_uniform_opt_forces_comp (pOSet, i, j, fij, &Eij);
 
-      if (isnan (Eij))
+      if (std::isnan (Eij))
         printf ("i = %d j = %d\n", i, j);
 
       Eij *= 0.5;
@@ -79,7 +79,7 @@
 
       net_ori_uniform_opt_forces_comp_neigh (pOSet, i, neighs[j], fij, &Eij, pqcloud);
 
-      if (isnan (Eij))
+      if (std::isnan (Eij))
         printf ("i = %d j = %d\n", i, j);
 
       Eij *= 0.5;
