--- src/mlpack/bindings/python/setup.py.in	2018-07-28 01:53:27.000000000 +0000
+++ src/mlpack/bindings/python/setup.py.in.new	2018-10-15 20:35:32.876452793 +0000
@@ -42,7 +42,7 @@
                   library_dirs=['${CMAKE_BINARY_DIR}/lib/'],
                   # CMAKE_CXX_FLAGS seems to have an extra space.
                   extra_compile_args=('-DBINDING_TYPE=BINDING_TYPE_PYX ' + \
-                      '-std=c++11${CMAKE_CXX_FLAGS}').split(' '),
+                      '-std=c++11 ${CMAKE_CXX_FLAGS}').split(' '),
                   extra_link_args=['${OpenMP_CXX_FLAGS}'],
                   undef_macros=[] if len("${DISABLE_CFLAGS}") == 0 \
                       else '${DISABLE_CFLAGS}'.split(';')) \
@@ -61,7 +61,7 @@
                   library_dirs=['${CMAKE_BINARY_DIR}/lib/'],
                   # CMAKE_CXX_FLAGS seems to have an extra space.
                   extra_compile_args=('-DBINDING_TYPE=BINDING_TYPE_PYX ' + \
-                      '-std=c++11${CMAKE_CXX_FLAGS}').split(' '),
+                      '-std=c++11 ${CMAKE_CXX_FLAGS}').split(' '),
                   extra_link_args=['${OpenMP_CXX_FLAGS}'],
                   undef_macros=[] if len("${DISABLE_CFLAGS}") == 0 \
                       else '${DISABLE_CFLAGS}'.split(';')) \
