--- tools/setup_helpers/cmake.py.org	2021-12-14 15:31:05.751856685 -0600
+++ tools/setup_helpers/cmake.py	2021-12-14 15:31:13.907840530 -0600
@@ -8,7 +8,7 @@
 from subprocess import check_call, check_output, CalledProcessError
 import sys
 import sysconfig
-from setuptools import distutils  # type: ignore[import]
+import distutils.version  # type: ignore[import]
 from typing import IO, Any, Dict, List, Optional, Union, cast
 
 from . import which
