--- Build.PL.old	2019-10-08 19:22:02.974565031 -0500
+++ Build.PL	2019-10-08 19:22:10.116561938 -0500
@@ -60,6 +60,6 @@
       },
     );
 
-$build->install_path( conf => $build->install_base ? File::Spec->catfile($build->install_base, 'conf') : "/etc/circleator/conf" );
+$build->install_path( conf => $build->install_base ? File::Spec->catfile($build->install_base, 'conf') : "/tmp/circleator/conf" );
 $build->add_build_element('cfg');
 $build->create_build_script;
--- bin/rasterize-svg.old	2019-10-08 19:23:40.540521524 -0500
+++ bin/rasterize-svg	2019-10-08 19:24:03.597511083 -0500
@@ -8,7 +8,7 @@
 ## globals
 my $USAGE = "Usage: $0 svg_file pdf|png|jpeg [output_width output_height]";
 # path to Apache Batik SVG rasterizer
-my $RASTERIZER = 'rasterizer';
+my $RASTERIZER = 'batik-rasterizer';
 my $DEFAULT_IMG_WIDTH = 4000;
 my $MAX_PDF_WIDTH = 2000;
 my $MTYPES = 
