--- src/cxx/lib/io/Xml.cpp	2017-06-22 12:14:50.000000000 -0500
+++ src/cxx/lib/io/Xml.cpp.new	2019-06-19 13:57:02.228863665 -0500
@@ -168,19 +168,11 @@
     if (!tree.empty())
     {
         unindex(*tree.begin(), treeWithIndexAttributes);
-#ifndef WIN32
-        boost::property_tree::write_xml(os, treeWithIndexAttributes, boost::property_tree::xml_writer_make_settings(' ', 2));
-#else
         boost::property_tree::write_xml(os, treeWithIndexAttributes, boost::property_tree::xml_writer_make_settings<std::string>(' ', 2));
-#endif
     }
     else
     {
-#ifndef WIN32
-        boost::property_tree::write_xml(os, tree, boost::property_tree::xml_writer_make_settings(' ', 2));
-#else
         boost::property_tree::write_xml(os, tree, boost::property_tree::xml_writer_make_settings<std::string>(' ', 2));
-#endif
     }
     return os;
 }
