--- opt/microsoft/ropen/3.5.3/lib64/pkgconfig/libR.pc	2018-08-05 02:40:18.000000000 +0000
+++ opt/microsoft/ropen/3.5.3/lib64/pkgconfig/libR.pc.new	2018-12-07 19:24:29.004051810 +0000
@@ -1,10 +1,10 @@
-rhome=/builddir/target/R/Linux/lib64/R
+rhome=/opt/anaconda1anaconda2anaconda3/lib/R
 rlibdir=${rhome}/lib
-rincludedir=/builddir/target/R/Linux/lib64/R/include
+rincludedir=${rhome}/include
 
 Name: libR
 Description: R as a library
 Version: 3.5.3
-Libs: -Wl,--export-dynamic -fopenmp  -L/builddir/vendor/build/lib -L${rlibdir} -lR
+Libs: -Wl,--export-dynamic -Wl,-rpath=${rlibdir} -Wl,-rpath=/opt/anaconda1anaconda2anaconda3/lib -fopenmp -L${rlibdir} -lR
 Cflags: -I${rincludedir} -I${rincludedir}
 Libs.private:
