From 14c4372eea35df4ea5faeca32a2b42904234e16d Mon Sep 17 00:00:00 2001
From: Eric Cooper <ecc@cmu.edu>
Date: Mon, 7 Mar 2011 21:48:33 -0500
Subject: [PATCH] fix assembly failure with binutils 2.21


Signed-off-by: Eric Cooper <ecc@cmu.edu>
---
 asmcomp/amd64/emit.mlp |   13 +++++++------
 1 files changed, 7 insertions(+), 6 deletions(-)

diff --git a/asmcomp/amd64/emit.mlp b/asmcomp/amd64/emit.mlp
index 55da47b..45afbb7 100644
--- a/asmcomp/amd64/emit.mlp
+++ b/asmcomp/amd64/emit.mlp
@@ -688,17 +688,18 @@ let fundecl fundecl =
   emit_all true fundecl.fun_body;
   List.iter emit_call_gc !call_gc_sites;
   emit_call_bound_errors ();
+  begin match Config.system with
+    "linux" | "gnu" ->
+      `	.type	{emit_symbol fundecl.fun_name},@function\n`;
+      `	.size	{emit_symbol fundecl.fun_name},.-{emit_symbol fundecl.fun_name}\n`
+    | _ -> ()
+  end;
   if !float_constants <> [] then begin
     if macosx
     then `	.literal8\n`
     else `	.section	.rodata.cst8,\"a\",@progbits\n`;
     List.iter emit_float_constant !float_constants
-  end;
-  match Config.system with
-    "linux" | "gnu" ->
-      `	.type	{emit_symbol fundecl.fun_name},@function\n`;
-      `	.size	{emit_symbol fundecl.fun_name},.-{emit_symbol fundecl.fun_name}\n`
-  | _ -> ()
+  end
 
 (* Emission of data *)
 
-- 
1.7.2.3

