--- src/mkdssp.cpp.org	2022-12-14 19:03:35.376697033 -0600
+++ src/mkdssp.cpp	2022-12-14 19:04:14.071626846 -0600
@@ -10,6 +10,7 @@
 #include "config.h"
 #endif
 
+#include <iostream>
 #include "dssp.h"
 #include "iocif.h"
 #include "mas.h"
--- src/mkhssp.cpp.org	2022-12-14 19:03:31.952703244 -0600
+++ src/mkhssp.cpp	2022-12-14 19:03:56.953657896 -0600
@@ -8,6 +8,7 @@
 #include "config.h"
 #endif
 
+#include <iostream>
 #include "fasta.h"
 #include "hssp-nt.h"
 #include "mas.h"
