--- Makefile.Cuda.org	2023-04-04 14:34:23.842532230 -0500
+++ Makefile.Cuda	2023-04-04 14:36:51.965125307 -0500
@@ -15,7 +15,7 @@
 CPP = g++
 UNAME := $(shell uname)
 
-TARGETS = 52 60 61 70
+TARGETS = 52 60 61 70 80 86 89 90
 CUDA_TARGETS=$(foreach target,$(TARGETS),-gencode arch=compute_$(target),code=sm_$(target))
 
 ifeq ($(DEVICE), CPU)
@@ -94,16 +94,16 @@
 
 ifeq ($(CONFIG),FDEBUG)
 	OPT =-O0 -g3 -Wall -DDOCK_DEBUG
-	CUDA_FLAGS = -G -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11
+	CUDA_FLAGS = -G -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11 -ccbin $(CC)
 else ifeq ($(CONFIG),LDEBUG)
 	OPT =-O0 -g3 -Wall
-	CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11
+	CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11 -ccbin $(CC)
 else ifeq ($(CONFIG),RELEASE)
 	OPT =-O3
-	CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11
+	CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11 -ccbin $(CC)
 else
 	OPT =
-	CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11
+	CUDA_FLAGS = -use_fast_math --ptxas-options="-v" $(CUDA_TARGETS) -std=c++11 -ccbin $(CC)
 endif
 
 # ------------------------------------------------------
