--- tensorflow_addons/tensorflow_addons.bzl.org	2022-10-19 17:08:34.733881226 -0500
+++ tensorflow_addons/tensorflow_addons.bzl	2022-10-19 19:55:46.130398385 -0500
@@ -7,11 +7,12 @@
         cuda_srcs = [],
         deps = [],
         cuda_deps = [],
-        copts = [],
+        copts = ["-Iexternal"],
         **kwargs):
     deps = deps + [
         "@local_config_tf//:libtensorflow_framework",
         "@local_config_tf//:tf_header_lib",
+        "@google//:headers",
     ]
 
     if cuda_srcs:
--- WORKSPACE.org	2022-10-19 16:55:29.211269055 -0500
+++ WORKSPACE	2022-10-19 20:07:17.498079410 -0500
@@ -43,3 +43,15 @@
 load("@org_tensorflow//tensorflow:workspace0.bzl", "tf_workspace0")
 
 tf_workspace0()
+
+new_local_repository(
+    name = "google",
+    path = "$PREFIX/include/google",
+    build_file_content = """
+package(default_visibility = ["//visibility:public"])
+cc_library(
+    name = "headers",
+    hdrs = glob(["**/*.h"]) + glob(["**/*.inc"])
+)
+"""
+)
