--- src/overlap.hpp	2018-02-26 02:27:33.000000000 -0600
+++ src/overlap.hpp.new	2018-04-03 22:54:09.743061870 -0500
@@ -12,6 +12,7 @@
 #include <string>
 #include <utility>
 #include <unordered_map>
+#include <stdlib.h>
 
 namespace bioparser {
     template<class T>
--- vendor/spoa/include/spoa/alignment_engine.hpp	2018-02-26 02:28:01.000000000 -0600
+++ vendor/spoa/include/spoa/alignment_engine.hpp.new	2018-04-03 22:53:33.008753975 -0500
@@ -11,6 +11,7 @@
 #include <memory>
 #include <vector>
 #include <utility>
+#include <stdlib.h>
 
 namespace spoa {
 
--- vendor/spoa/src/graph.cpp	2018-02-26 02:28:01.000000000 -0600
+++ vendor/spoa/src/graph.cpp.new	2018-04-03 23:05:11.321596962 -0500
@@ -9,6 +9,7 @@
 #include <stack>
 
 #include "spoa/graph.hpp"
+#include <stdlib.h>
 
 namespace spoa {
 
