--- setup.py.org	2021-06-18 03:03:35.172256850 +0000
+++ setup.py	2021-06-18 03:04:29.615675405 +0000
@@ -348,7 +348,7 @@
         ffmpeg_include_dir = os.path.join(ffmpeg_root, 'include')
         ffmpeg_library_dir = os.path.join(ffmpeg_root, 'lib')
 
-        gcc = distutils.spawn.find_executable('gcc')
+        gcc = distutils.spawn.find_executable('x86_64-conda-linux-gnu-cc')
         platform_tag = subprocess.run(
             [gcc, '-print-multiarch'], stdout=subprocess.PIPE)
         platform_tag = platform_tag.stdout.strip().decode('utf-8')
