--- tensorflow/tensorflow.bzl	2018-01-25 22:22:10.000000000 +0000
+++ tensorflow/tensorflow.bzl.new	2018-02-02 21:51:15.039443161 +0000
@@ -264,7 +264,7 @@
     name,
     srcs=[],
     deps=[],
-    linkopts=[],
+    linkopts=['-lrt'],
     framework_so=tf_binary_additional_srcs(),
     **kwargs):
   native.cc_binary(
@@ -1264,7 +1264,7 @@
 )
 
 def tf_extension_linkopts():
-  return []  # No extension link opts
+  return ["-lrt"]  # No extension link opts
 
 def tf_extension_copts():
   return []  # No extension c opts
