--- qiime/support_files/denoiser/FlowgramAlignment/Makefile	2015-05-26 14:14:55.000000000 -0500
+++ qiime/support_files/denoiser/FlowgramAlignment/Makefile.new	2015-07-14 15:44:55.768281579 -0500
@@ -3,7 +3,7 @@
 all: FlowgramAli_4frame
 
 FlowgramAli_4frame: FlowgramAli_4frame.lhs
-	ghc --make -O2 FlowgramAli_4frame
+	ghc --make -O2 -L${PREFIX}/lib FlowgramAli_4frame 
 	cp FlowgramAli_4frame ../../../../scripts/
 # On ghc7+ consider using -with-rtsopts="-H100M" for increased speed
 
