--- Makefile.in	2013-08-14 20:22:40.000000000 +0000
+++ Makefile.in.new	2018-10-12 16:42:16.226632164 +0000
@@ -316,7 +316,7 @@
 MAINTAINERCLEANFILES = Makefile.in aclocal.m4 configure config.h.in	\
 config.log config.status stamp-h.in $(AUX_DIST)
 
-SUBDIRS = doc lib tests
+SUBDIRS = doc lib
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libmatheval.pc
 all: config.h
@@ -863,7 +863,7 @@
 .PHONY: beauty dist-hook
 
 beauty:
-	-for dir in lib tests; do cd $$dir; $(MAKE) $@; cd ..; done
+	-for dir in lib ; do cd $$dir; $(MAKE) $@; cd ..; done
 
 dist-hook:
 	for file in $(AUX_DIST); do \
